### PR TITLE
Webhooks

### PR DESCRIPTION
**Pull request by mick-roper**
* added a reference
* added a guide
* added a menu link

### CODE REVIEW

File: src/data/menu-guides.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@
 - title: Send custom indicators
   link: /guides/slo/sending-custom-indicators/
   sub: true
+- title: Registering webhooks
```
Comment (ciaranevans) [inline src/data/menu-guides.yaml:16]: Should this be:

```suggestion
- title: Registering Webhooks
```

As we capitalise it in the actual title of the post?

File: docs/reference/entities/webhookcriteria.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+A `WebhookCriteria` declaratively defines the data required to invoke a remote webhook when a certain internal event is received.
+
+## Structure
+
+```yaml
+apiVersion: reliably.com/v1
+kind: WebhookCriteria
+metadata:
+    labels: {}
+    relatedTo: {}
```
Comment (ciaranevans) [inline docs/reference/entities/webhookcriteria.md:19]: Is this required? Should we mark optional fields as such?

**Review by ciaranevans — CHANGES_REQUESTED** [review]

**Comment by ciaranevans** [comment]
Is the new lockfile intentional?

File: src/data/menu-guides.yaml (hunk 1 above)
Comment (mick-roper) [reply src/data/menu-guides.yaml:16]: Yep

File: docs/reference/entities/webhookcriteria.md (hunk 2 above)
Comment (mick-roper) [reply docs/reference/entities/webhookcriteria.md:19]: It's a good question - right now we don't describe things as optional or required in the examples. Might be something to think about though.

**Comment by mick-roper** [comment]
Nope its not - shouldn't make too much of a difference though with this project; its already got 22 serious vulnerabilities!!

**Review by ciaranevans — APPROVED** [review]

**Pull request merged**